scripts/Start-CleanMachineTesting.ps1

```diff
@@ -0,0 +1,33 @@
+<#
+.SYNOPSIS
+	Installs Windows Sandbox feature if needed and then sets up a clean machine test enviroment
+.DESCRIPTION
+	Starts Windows Sanbox instance and runs a powershell script to setup dependencies and a repo to test gitextensions on a clean machine.
+.NOTES
+	See https://learn.microsoft.com/en-us/windows/security/application-security/application-isolation/windows-sandbox/windows-sandbox-overview
+	to learn about windows sandbox. 
+	Caveat: You must have ran a build for this to fully work.  If artifacts folder is clean, there will be errors.
+#>
+
+#required to turn on windows feature
+#requires -RunAsAdministrator
+function StartWSB() {
+	#Include PSR( problem steps recorder)
+	Copy-Item  "$env:SystemRoot\system32\psr.exe" "$PSScriptRoot\wsb\"
+	Start-Process .\Sandbox.wsb
+}
+
+$test = (Get-WindowsOptionalFeature -Featurename "Containers-DisposableClientVM" -Online)
+if (!$test) {
+	Write-Warning "Unsupported Operating System: Windows 10 Pro or Enterprise 1903 or greater required."
+	throw
+}
+if ($test.state -eq 'Enabled') {
+	Write-Host "  ** Sandbox already installed!" -Foreground Magenta
+	StartWSB
+}
+else {
+	Enable-WindowsOptionalFeature -Online -FeatureName Containers-DisposableClientVM -NoRestart
```

Comment:
(My system needed to restart.)

Reply:
Ok. I'll add to documentation that it attempts to not require a restart but may require one.